networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,83 @@ These release notes track the development of the Network Observability Operator
+You can now collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device. For more information, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-SR-IOV-config_network_observability[Configuring the monitoring of SR-IOV interface traffic].
+
+[id="IPFIX-support-1.4"]
+==== IPFIX exporter support
+You can now export eBPF-enriched network flows to IPFIX. For more information, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-enriched-flows_network_observability[Export enriched network flow data].
+
+[id="network-observability-operator-1.4.0-bug-fixes"]
+=== Bug fixes
+* Previously, the Prometheus metrics exported by Network Observability were computed out of potentially duplicated network flows. In the related dashboards, from *Observe* -> *Dashboards*, this could result in potentially doubled rates. Note that dashboards from the *Network Traffic* view were not affected. Now, network flows are filtered to eliminate duplicates prior to metrics calculation, which results in correct traffic rates displayed in the dashboards. (link:https://issues.redhat.com/browse/NETOBSERV-1131[*NETOBSERV-1131*])
+
+* Previously, the Network Observability Operator agents were not able to capture traffic on network interfaces when configured with Multus or SR-IOV, non-default network namespaces. Now, all available network namespaces are recognized and used for capturing flows, allowing capturing traffic for SR-IOV. There are xref:../../networking/network_observability/configuring-operator.adoc#network-observability-SR-IOV-config_network_observability[configurations needed] for the `FlowCollector` and `SRIOVnetwork` custom resource to collect traffic. 
+(link:https://issues.redhat.com/browse/NETOBSERV-1283[*NETOBSERV-1283*])
+
+* Previously, in the Network Observability Operator details from *Operators* -> *Installed Operators*, the `FlowCollector` *Status* field might have reported incorrect information about the state of the deployment. The status field now shows the proper conditions with improved messages. The history of events is kept, ordered by event date. (link:https://issues.redhat.com/browse/NETOBSERV-1224[*NETOBSERV-1224*])
+
+* Previously, during spikes of network traffic load, certain eBPF pods were OOM-killed and go into `CrashLoopBackOff` state with the 429 error: 'Per stream rate limit exceeded (limit:xMB/sec) while attempting to ingest for stream'. Now, you can xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-troubleshooting-loki-tenant-rate-limit_network-observability-troubleshooting[configure the `LokiStack` custom resource `spec.limits.global.ingestion`] with the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications restart the pods in your cluster and stop the 429 rate-limit error. (link:https://issues.redhat.com/browse/NETOBSERV-975[*NETOBSERV-975*]) 
```

Comment:
I think this is mixing several things: the "429 error: Per stream rate limit exceeded" isn't thrown by the ebpf agents - and the new LokiStack config was tracked in https://issues.redhat.com/browse/NETOBSERV-717 which we didn't include in the release note because it's a loki-operator change, not netobserv.
We may reformulate to focus strictly on the eBPF agent memory improvement that we did to avoid OOM-killing / CrashLoopBackOff

Reply:
Ah ok...I'm not sure how I got this mixed up but thanks for clarifying the difference.

Reply:
probably because the original JIRA ticket was also mixing those things :)